tools/ci_build/github/linux/build_cuda_c_api_package.sh

```diff
@@ -2,4 +2,4 @@
 set -e -x
 docker run -e SYSTEM_COLLECTIONURI --rm --volume \
 $BUILD_SOURCESDIRECTORY:/onnxruntime_src --volume $BUILD_BINARIESDIRECTORY:/build -e NIGHTLY_BUILD onnxruntimecuda${CUDA_VERSION_MAJOR}build \
-/bin/bash -c "/usr/bin/python3 /onnxruntime_src/tools/ci_build/build.py --enable_lto --build_java --build_nodejs --build_dir /build --config Release --skip_submodule_sync  --parallel --use_binskim_compliant_compile_flags --build_shared_lib --use_cuda --cuda_version=$CUDA_VERSION --cuda_home=/usr/local/cuda-$CUDA_VERSION --cudnn_home=/usr/local/cuda-$CUDA_VERSION --skip_tests --use_vcpkg --cmake_extra_defines 'CMAKE_CUDA_ARCHITECTURES=75-real;80-real;90' && cd /build/Release && make install DESTDIR=/build/installed"
+/bin/bash -c "/usr/bin/python3 /onnxruntime_src/tools/ci_build/build.py --enable_lto --build_java --build_nodejs --build_dir /build --config Release --skip_submodule_sync  --parallel --use_binskim_compliant_compile_flags --build_shared_lib --use_cuda --cuda_version=$CUDA_VERSION --cuda_home=/usr/local/cuda-$CUDA_VERSION --cudnn_home=/usr/local/cuda-$CUDA_VERSION --skip_tests --use_vcpkg --cmake_extra_defines 'CMAKE_CUDA_ARCHITECTURES=61-real;70-real;75-real;80-real;90' && cd /build/Release && make install DESTDIR=/build/installed"
```

Comment:
Suggest to use 60-real instead of 61-real. I think 60-real could work on 61, but 61-real might not run on 60.

Tesla P100 | 6.0
Tesla P40 | 6.1
Tesla P4 | 6.1

Reply:
If wheel size allowed, add 86-real that could help a few popular data center GPU like:
NVIDIA L4 | 8.9
NVIDIA L40 | 8.9
NVIDIA A40 | 8.6
NVIDIA A10 | 8.6
NVIDIA A16 | 8.6
NVIDIA A2 | 8.6

Reply:
Thanks! `60-real` just applied